hunting/generate_markdown.py

```diff
@@ -50,7 +50,9 @@ def load_all_toml(base_path: Path) -> List[tuple[Hunt, Path]]:
     """Load all TOML files from the directory and return a list of Hunt configurations and their paths."""
     hunts = []
     for toml_file in base_path.rglob("*.toml"):
-        hunt_config = load_toml(toml_file.read_text())
+        with toml_file.open('r', encoding='utf-8') as file:
```

Comment:
```suggestion
        hunt_config = load_toml(toml_file.read_text(encoding='utf-8'))
```

Reply:
Did we try this? Just curious why we reverted to open file and read mode when we could pass encoding to the read_text option.

Reply:
Great observation, reverted and made the minor tweak!